test/e2e/specs/editor/various/fullscreen-mode.spec.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Full Screen', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test.afterEach( async ( { requestUtils } ) => {
+		await requestUtils.deleteAllPosts();
+	} );
+
+	test( 'should open the fullscreen mode from the more menu', async ( {
+		page,
+	} ) => {
+		// Open Options
+		await page.locator( ' [aria-label="Options"]' ).click();
+
+		// Verify All options are visible
+		page.locator( '.interface-more-menu-dropdown__content' );
+
+		// Choose Full Screen Mode
+		await page
+			.locator( 'text=/Fullscreen modeWork without distraction/' )
+			.click();
+
+		// Validate Fullscreen
+		const isFullscreenEnabled = await page.$eval( 'body', ( body ) => {
+			return body.classList.contains( 'is-fullscreen-mode' );
+		} );
+
+		expect( isFullscreenEnabled ).toBe( true );
```

Comment:
```suggestion
		// Check the body class.
		await expect( page.locator( 'body' ) ).toHaveClass(
			/is-fullscreen-mode/
		);
```

Suggestion: Let's use native API for class assertions - https://playwright.dev/docs/test-assertions#locator-assertions-to-have-class.